rca/people/models.py

```diff
@@ -90,6 +125,9 @@ class StaffPage(BasePage):
             ],
             heading="Details",
         ),
+        MultiFieldPanel(
+            [InlinePanel("role", label=_("Staff role"))], heading=_("Staff roles")
+        ),
```

Comment:
Is the MultiFieldPanel necessary here? I always thought InlinePanel looked fine as a top-level item.

Reply:
removed it now since we have a label specified